src/Data/Maybe/Base.agda

```diff
@@ -137,3 +132,15 @@ thisM a = maybe′ (these a) (this a)
 
 thatM : Maybe A → B → These A B
 thatM = maybe′ these that
+
+------------------------------------------------------------------------
+-- Deprecated
+
+-- Version 2.1
+-- decToMaybe
+
+decToMaybe = Relation.Nullary.Decidable.Core.decToMaybe
```

Comment:
To be honest, I think the easiest approach which would avoid all the import cycle comments would just be to entirely redefine it here. It's pretty short...

Reply:
Sure, can do.

Reply:
Doesn't actually help: the direct definition still need `Relation.Nullary.Decidable.Core` to be imported.